### DISCUSSION TITLE
Responsiveness issue ( dimensions ) in creating new item interface

### DISCUSSION THREAD

**Discussion by fballan93** [directus/directus#4978] [Archive]
when using retina displays or high resolution displays the interface doesn't fill the display 

<img width="1678" alt="CleanShot 2021-04-11 at 11 29 17@2x" src="https://user-images.githubusercontent.com/9093101/114297519-c33b4200-9ab9-11eb-8a93-4798c567276f.png">

my environment:
directus v9.0.0-rc.57
DBMS : (MySQL 8)
deployment mode : npx
Browser : Chrome and Edge

**Comment by benhaynes** [comment]
Not sure what this ticket is meant to ask. The form uses two fields side by side, and those fields have a max width. They get unwieldy when too wide, and we can't shift the whole form to wrap other fields up because the layout is set by the admin.

The only visual thing we could do is center the form on the page, but that would mess with alignment. Do you have any recommendations?

**Reply by benhaynes** [reply]
Also, you _can_ set fields to "fill" to use that space as needed!

**Reply by fballan93** [reply]
the current grid in the layout is working as 50% per item. the thing is that when you collapse the right drawer the middle layout display do not expand. 

"Also, you can set fields to "fill" to use that space as needed!" i agree with you and i tried it the thing is fill entire width will force one item per row and still not being responsive . i tried changing my screen dimensions to smaller one the same behavior applies 

![CleanShot 2021-04-12 at 13 12 04](https://user-images.githubusercontent.com/9093101/114378823-c2c1aa80-9b90-11eb-953c-337140fd194f.gif)

**Reply by benhaynes** [reply]
Hmm, still not sure about the exact issue you're trying to report. The fields (and there fore interfaces) all have a min/max width:

```
--form-column-width: 300px;
--form-column-max-width: 380px;
```

So regardless of the page's width, those fields won't expand beyond 380px when in "half" mode, with a 32px margin between. So your page width looks like this:

`32px + 380px + 32px + 380px + 32px` = 856px total

Anything beyond this will appear empty, and this is intended behavior.
